File: java-dsa/src/general/Palindrome.java
public class Palindrome {
    public static void main(String[] args) {
        int n = 969;
        int original =n;
        int remainder = 0;
        int rev = 0;
        while (n > 0) {
            remainder = n % 10;
            rev = rev * 10 + remainder;
            n = n / 10;

        }
        if(original==rev){
            System.out.println("Palindrome");
        }
            else{
                System.out.println("Not a palindrome number");
            }
        }
    }

File: javascript-dsa/Palindrome.js
let n = 979;
let original = n;
let rev = 0;
let digit = 0;
while (n != 0) {
  digit = n % 10;
  rev = rev * 10 + digit;
  n = Math.floor(n / 10);
}

if (original == rev) {
  console.log("Palindrome");
} else {
  console.log("Not a palindrome number");
}
